src/attributes.js

```diff
@@ -122,19 +123,20 @@ var defaults = {
  * A publicly mutable object to provide custom mutators for attributes.
  * @const {!Object<string, function(!Element, string, *)>}
  */
-var mutators = {
+var attributes = {
   // Special generic mutator that's called for any attribute that does not
   // have a specific mutator.
   [symbols.all]: applyAttributeTyped,
 
   // Special case the style attribute
-  'style': applyStyle
+  'style': applyStyle,
+
+  [symbols.defaults]: defaults
```

Comment:
So usage might look something like:

```
import { attributes, symbols } from 'incremental-dom';
attributes.innerHTML = attributes[symbols.defaults].applyProp;
```

which feels a bit weird.

Perhaps something like:

```
attributes.innerHTML = attributes[symbols.applyTyped];
```

might make more sense. I think going back to exposing the defaults as a separate object (maybe applicators? appliers?) would probably be the best idea though. The attributes file can export both.